pkg/rpcserver/runner.go

```diff
@@ -320,24 +408,43 @@ func (runner *Runner) sendSignalUpdate(plus []uint64) error {
 			},
 		},
 	}
-	return flatrpc.Send(runner.conn, msg)
+	return flatrpc.Send(conn, msg)
 }
 
-func (runner *Runner) sendCorpusTriaged() error {
+func (runner *Runner) SendCorpusTriaged() error {
+	conn := runner.conn
+	if conn == nil {
```

Comment:
It will crash with almost as readable panic w/o this additional check.

I think it makes sense to add additional code/check only when (1) we would not detect the bug otherwise, or (2) we want to print more information that would be missing otherwise.
If we detect it anyway and have the same info, then it's just additional code, and there thousands of such potential checks we could, basically for any pointer use.